packages/dependency-extraction-webpack-plugin/src/index.js

```diff
@@ -42,6 +43,7 @@ const wooRequestToHandle = ( request ) => {
 		const handle = request.substring( WOOCOMMERCE_NAMESPACE.length );
 		const irregularHandleMap = {
 			data: 'wc-store-data',
+			'block-data': 'wc-blocks-data-store',
```

Comment:
Same here. If the package name is `blocks-data`, can the handle simply be `wc-blocks-data`?

Reply:
The handle we have is `wc-blocks-data-store`, so I don't think this can be changed without some backward compat.

Again, this thing predates me, so I don't know the reasons behind the naming, just that changing them now is a breaking change, given that we recommend this plugin to 3PD